packages/ui/react-ui/package.json

```diff
@@ -28,8 +27,8 @@
         "version": "yarn run create-version-file && git add .",
         "prebuild": "yarn run create-version-file",
         "clean": "shx rm -rf lib/*",
-        "build": "yarn clean && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json",
-        "postbuild": "echo '{\"type\":\"commonjs\"}' | npx json > lib/cjs/package.json && echo '{\"type\":\"module\"}' | npx json > lib/esm/package.json",
+        "build": "yarn clean && tsc -p tsconfig.json && tsc-esm -p tsconfig.json && tsc -p tsconfig.cjs.json",
+        "postbuild": "echo '{\"type\":\"commonjs\"}' | npx json > lib/cjs/package.json && echo '{\"type\":\"module\"}' | npx json > lib/esm/package.json && shx mv lib/esm/index.js lib/esm/index.mjs",
```

Comment:
Do we actually need the `shx mv lib/esm/index.js lib/esm/index.mjs` part? Its failing on my end:

`mv: no such file or directory: lib/esm/index.js`

Looks like index.mjs is already being generated.

Reply:
uhh its 100% not being generated on my machine, meaning the mv command is required